aiohttp/helpers.py

```diff
@@ -71,12 +73,77 @@
         dataclasses.dataclass, frozen=True, slots=True
     )
 
-__all__ = ("BasicAuth", "ChainMapProxy", "ETag", "frozen_dataclass_decorator", "reify")
+__all__ = (
+    "BasicAuth",
+    "ChainMapProxy",
+    "ETag",
+    "frozen_dataclass_decorator",
+    "make_non_quoted_morsel",
+    "make_quoted_morsel",
+    "preserve_morsel_with_coded_value",
+    "reify",
```

Comment:
not a good idea, someone will think this is ok for public use. Let me move them all to a protected namespace